internal/cli/iam/users/describe.go

```diff
@@ -0,0 +1,102 @@
+// Copyright 2020 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package users
+
+import (
+	"errors"
+
+	"github.com/mongodb/mongocli/internal/cli"
+	"github.com/mongodb/mongocli/internal/config"
+	"github.com/mongodb/mongocli/internal/flag"
+	"github.com/mongodb/mongocli/internal/store"
+	"github.com/mongodb/mongocli/internal/usage"
+	"github.com/spf13/cobra"
+)
+
+const describeTemplate = `ID	FIRST NAME	LAST NAME	USERNAME
+{{.ID}}	{{.FirstName}}	{{.LastName}}	{{.Username}}
+`
+
+type DescribeOpts struct {
+	cli.GlobalOpts
+	cli.OutputOpts
+	store    store.UsersDescriber
+	username string
+	ID       string
+}
+
+func (opts *DescribeOpts) init() error {
+	var err error
+	opts.store, err = store.New(config.Default())
+	return err
+}
+
+func (opts *DescribeOpts) Run() error {
+	var r interface{}
+	var err error
+
+	if opts.username != "" {
+		r, err = opts.store.UserByName(opts.username)
+	}
+
+	if opts.ID != "" {
+		r, err = opts.store.UserByID(opts.ID)
+	}
+
+	if err != nil {
+		return err
+	}
+
+	return opts.Print(r)
+}
+
+func (opts *DescribeOpts) validate() error {
+	if opts.ID == "" && opts.username == "" {
+		return errors.New("must supply one of 'ID' or 'username'")
```

Comment:
[nit] I guess the flag is lower case, so we should make it so here
```suggestion
		return errors.New("must supply one of 'id' or 'username'")
```